src/main/java/de/adorsys/keycloak/config/provider/KeycloakImportProvider.java

```diff
@@ -45,16 +45,17 @@
 public class KeycloakImportProvider {
     private static final Logger logger = LoggerFactory.getLogger(KeycloakImportProvider.class);
 
-    private static final StringSubstitutor interpolator = StringSubstitutor.createInterpolator()
-            .setEnableSubstitutionInVariables(true)
-            .setEnableUndefinedVariableException(true);
+    private final StringSubstitutor interpolator;
 
     private final ImportConfigProperties importConfigProperties;
 
     public KeycloakImportProvider(
             ImportConfigProperties importConfigProperties
     ) {
         this.importConfigProperties = importConfigProperties;
+        interpolator = StringSubstitutor.createInterpolator()
+                .setEnableSubstitutionInVariables(importConfigProperties.isVarSubstitutionInVariables())
+                .setEnableUndefinedVariableException(importConfigProperties.isVarSubstitutionUndefinedThrowsExceptions());
```

Comment:
Could this cover by a test that ensures if parameter changes the behavior of the StringSubstitutor?

A test looks be pointless here, but I every tiny feature should cover by a test. Just in case if I do some refactor. Tests could help me to ensure the defined functionality.